mkdocs_monorepo_plugin/parser.py

```diff
@@ -41,7 +43,22 @@ def __loadAliasesAndResolvedPaths(self, nav=None):
             if type(item) is str:
                 value = str
             elif type(item) is dict:
+                key = list(item.keys())[0]
                 value = list(item.values())[0]
+                if key.startswith(WILDCARD_INCLUDE_STATEMENT) and value.startswith(INCLUDE_STATEMENT):
+                    root_dir = Path(self.config['config_file_path']).parent
+                    mkdocs_path = value[len(INCLUDE_STATEMENT):]
+                    base_path, *name = key[len(WILDCARD_INCLUDE_STATEMENT):].split()
+                    search_path = Path(base_path) / mkdocs_path
+                    key = " ".join(name)
+                    dirs = sorted(root_dir.glob(str(search_path)))
```

Comment:
On the one hand, I like the simplicity of just using `glob`, but on the other hand, that opens up quite a few possibilities, some of which may or may not be desirable?  e.g. `**` could potentially be slow in very large monorepos, and maybe `?` could be useful, but character ranges `[]` less so?

I mostly worry because the glob syntax would form an implicit API going forward and maybe a smaller surface area that covers the most obvious examples would be better to start (e.g. `mkdcs.yml` files under a single wildcard directory).

Reply:
I left the `glob` behavior unchanged for now. I definitely see your point about some features being unnecessary and the potentially slow performance of `**` on large monorepos. I would argue that, in addition to the simplicity of using `glob`, users may expect to be able to use the full range of syntax options and be aware of the performance pitfalls of recursive globbing. That said, if you feel that a more basic implementation would be preferred, let me know and I can implement it.

Reply:
Yeah, it makes sense.  Perhaps a test case to cover other aspects of `glob` formatting (maybe just `**`) would be sufficient?

Reply:
I added a test case for `**` in https://github.com/backstage/mkdocs-monorepo-plugin/pull/67/commits/55c4a0e83fa97891955be1c2b22c15f3774cb107